uaclient/security_status.py

```diff
@@ -156,6 +157,14 @@ def security_status(cfg: UAConfig) -> Dict[str, Any]:
             }
         )
 
+    summary["num_installed_packages_main"] = package_count["main"]
```

Comment:
`num_installed_packages_main` follows a different naming pattern from the currently existing `num_esm_infra_packages` - should we change these to `num_main_packages` and similar for all new fields?

Reply:
I'd go with the opposite, something like `num_installed_packages_esm_infra`

Reply:
We can do that, and it would also be a major schema change, but if we're doing a major schema upgrade, then we might as well make any changes that we want to at once.